__tests__/app.test.js

```diff
@@ -33,7 +33,31 @@ describe("app tests", () => {
       test("200: responds with review object passed by user", () => {
         const REVIEW_ID = 1;
         return request(app)
-          .get(`/api/reviews/${REVIEW_ID}`)
+          .get(`/api/reviews/1`)
+          .expect(200)
+          .then(({ body: { review } }) => {
+            expect(review).toEqual({
+              review_id: 1,
+              title: "Agricola",
+              designer: "Uwe Rosenberg",
+              owner: "mallionaire",
+              review_img_url:
+                "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+              review_body: "Farmyard fun!",
+              category: "euro game",
+              created_at: "2021-01-18T10:00:20.514Z",
+              votes: 1,
+            });
+          });
+      });
+    });
+    describe("PATCH /api/reviews/:review_id", () => {
+      test("200: responds with updated review object where vote is incremented by newVote value", () => {
+        const REVIEW_ID = 1;
```

Comment:
great use of `const` variables in tests

you should use this to match the `review_id` in the returned review too!

```diff
{
- review_id: 1,
+ review_id: REVIEW_ID,
}
```

🌟